docs/src/app/components/pages/components/List/ExampleChat.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import MobileTearSheet from '../../../MobileTearSheet';
+import Avatar from 'material-ui/lib/avatar';
+import List from 'material-ui/lib/lists/list';
+import ListItem from 'material-ui/lib/lists/list-item';
+import Divider from 'material-ui/lib/divider';
+import CommunicationChatBubble from 'material-ui/lib/svg-icons/communication/chat-bubble';
+
+const ListExampleChat = () => (
+  <MobileTearSheet>
+    <List subheader="Recent chats">
+      <ListItem
+        primaryText="Brendan Lim"
+        leftAvatar={<Avatar src="images/ok-128.jpg" />}
+        rightIcon={<CommunicationChatBubble />} />
```

Comment:
`<CommunicationChatBubble />` is repeated 6 times. The page is quite slow, can we create a
```
const communicationChatBubble = < CommunicationChatBubble/>;
```
outside of the render method?